src/phylo/phyplot.jl

```diff
@@ -0,0 +1,285 @@
+@enum nodesym circle=1 square=2 rect=3
+@enum textpos auto=1 alignright=2 alignleft=3 above=4
+@enum plottype phylogram=1 fan=2
+
+type NodeFormat#{S<:AbstractString}
```

Comment:
Yes, exactly - that is because the NodeAnnotation{S} fields are also commented (but maybe these should just be ```UTF8String```s, I am unsure whether it is better to have a parametric type?). I want the NodeFormat type to have an active constructor that allows the type to be created with a range of arguments.